qiskit_experiments/test/Fine Calibration Tutorial.rst

```diff
@@ -0,0 +1,154 @@
+================================================
+Fine Calibrations with Pulse Simulator
+================================================
+The amplitude of a pulse can be precisely calibrated using
+error amplifying gate sequences. These gate sequences apply 
+the same gate a variable number of times. Therefore, if each gate
+has a small error dtheta  in the rotation angle then 
+a sequence of n gates will have a rotation error of n*dtheta.
+
+Since our new pulse simulator cache the calibrations that it has already solved
+in the form of dictionary in which keys are a tuple of instruction name, qubits and parameters
+and the vaule is a unitary matrix, it is efficient when simulating a circuit
+which consists of same repeated gates. 
+Therefore, fine amplitude calibration experiment which is a typical example of
+containing such repeated gates within a circuit, can be run efficiently in the "IQPulseBackend"
+
+Let's start with importing some necessary modules and build our pulse simulator with
+"SingleTransmonTestBackend" which is inherited from "IQPulseBackend"
+
+.. jupyter-execute:: 
+
+    import numpy as np
+    from qiskit.pulse import InstructionScheduleMap
+    import qiskit.pulse as pulse
+    from qiskit_experiments.library import FineXAmplitude, FineSXAmplitude
+    from qiskit_experiments.test.iq_pulse_backend import SingleTransmonTestBackend
+
+.. jupyter-execute::
+
+    backend = SingleTransmonTestBackend()
+    qubit = 0
+-----------------------------------------------------
+Fine X gate Amplitude Calibration
+-----------------------------------------------------
+We will run the fine calibration experiments with our own pulse schedules. 
+To do this we create an instruction to schedule map which we populate with 
+the schedules we wish to work with. This instruction schedule map is then 
+given to the transpile options of the calibration experiments so that 
+the Qiskit transpiler can attach the pulse schedules to the gates in the experiments. 
+We will base all our pulses on the default X pulse of "SingleTransmonTestBackend".
+
+.. jupyter-execute::
+    x_pulse = backend.defaults().instruction_schedule_map.get('x', (qubit,)).instructions[0][1].pulse
+    d0, inst_map = pulse.DriveChannel(qubit), InstructionScheduleMap()
+
+    for name, pulse_ in [("x", x_pulse)]:
+        with pulse.build(name=name) as sched:
+            pulse.play(pulse_, d0)
+
+        inst_map.add(name, (qubit,), sched)
```

Comment:
This is a bit confusing. I would only add one pulse to the inst. map. Namely the one with the over or under rotation.

Reply:
```suggestion
```